src/main/java/de/ait/secondlife/domain/dto/OfferCreationDto.java

```diff
@@ -0,0 +1,44 @@
+package de.ait.secondlife.domain.dto;
+
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+@Schema(description = "Offer create DTO")
+public class OfferCreationDto {
+    @Schema(description = "Title of a offer", example = "Upholstered chair")
+    private String title;
+    @Schema(description = "Description of a offer",
+            example = "In a small village, a curious cat named Whiskers explored every nook " +
+                    "and cranny. One day, Whiskers discovered a hidden garden filled with vibrant " +
+                    "flowers and buzzing bees. The villagers often saw Whiskers napping among the " +
+                    "blossoms, bringing a smile to everyone's face. Whiskers " +
+                    "became the beloved guardian of the magical garden.\n")
+    private String description;
+    @Schema(description = "Auction duration in days", example = "3")
+    private Integer auctionDurationDays;
+    @Schema(description = "Starting offer price", example = "1234.34")
+    private BigDecimal startPrice;
+    @Schema(description = "Bidding step", example = "12.34")
+    private BigDecimal step;
+    @Schema(description = "Possible buyout price without bidding", example = "1222.34")
+    private BigDecimal winBid;
+    @Schema(description = "Offer is free or not", example = "true")
+    private Boolean isFree;
+    @Schema(description = "User id of the owner of the offer", example = "34")
+    private Long userId;
```

Comment:
userId should not be in Dto. we will get it from JWT for authenticated users

Reply:
This is a temporary solution. After the codes are merged, it is necessary to make a task for it. As only the owner of the offer can do it. also when changing it, it is necessary to check that the ownerId and the id of the current user coincide.